README.md

```diff
@@ -259,7 +259,7 @@ This comparison between `$outputContext.Data` vs `$outputContext.PreviousData` i
 
 Debugging is _arguably_ one of the most complex topics in any programming / scripting language.
 
-The templates also comes with a __debugStart.ps1__, __config.json__ and __demoPerson.json__ in the _test_ folder. They allows you to easily debug your scripts. You can _mock_ variables such as the `$personContext`, `$actionContext` and all built-in variables in HelloID you need. By mocking these variables, you can easily test your scripts under a variety of conditions, without having to worry about external dependencies or data sources.
+The templates also comes with a __debugStart.ps1__, __config.json__, __actionContext.json__ and __demoPerson.json__ in the _test_ folder. They allows you to easily debug your scripts. You can _mock_ variables such as the `$personContext`, `$actionContext` and all built-in variables in HelloID you need. By mocking these variables, you can easily test your scripts under a variety of conditions, without having to worry about external dependencies or data sources.
 
```

Comment:
ik denk dat we deze tekst toch even moeten laten staan. Zij het dat dan ipv bv `actionContext` bv `resourceContext` pakken. Want niet alle variables zitten in de debugStart.

```
You can _mock_ variables such as the `$personContext`, `$actionContext` and all built-in variables in HelloID you need. By mocking these variables, you can easily test your scripts under a variety of conditions, without having to worry about external dependencies or data sources.
```

Reply:
ik heb nu de namen 1 op 1 overgenomen met wat er in de test folder zit. Lijkt mij ok.